.changeset/thirty-kiwis-trade.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/admin.server-configurations.v1": major
+"@wso2is/console": major
```

Comment:
Update to minor/patch

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/7004/commits/7c573064160ca9f7a3845891ff1ce5b8ccfb6700

Reply:
Let's delete the changeset with major and only keep the one with minor version update.

Reply:
Updated by https://github.com/wso2/identity-apps/pull/7004/commits/59d040047dd8b21efd511e781c167e8ebd65b64f